src/server/db_slice.cc

```diff
@@ -33,6 +34,8 @@ ABSL_FLAG(double, table_growth_margin, 0.4,
           "Prevents table from growing if number of free slots x average object size x this ratio "
           "is larger than memory budget.");
 
+ABSL_FLAG(bool, expiration_keyspace_events, false, "Send keyspace events for expiration");
```

Comment:
to what setting is it equivalent for `notify-keyspace-events`
https://redis.io/docs/latest/develop/use/keyspace-notifications/#configuration

Reply:
Yes, but it's configured with it's own flag syntax, so not sure if we take over it, just keep the name with a boolean flag or use a differently named flag

Reply:
my question is what is the setting of `notify-keyspace-events` in valkey should be to roughly be equivalent to what you implemented here. How do you explain this feature to a valkey user?

Reply:
Setting true is equvalent to using Ke in notify-keyspace-events

Reply:
Ok, please add it to the help string

Reply:
what about 'x' ? we do not handle it?

Reply:
Eviction? No, we don't, but we can if needed

Reply:
'x' is expiry and 'e' is eviction. So which one we support?  :)
is it `Ke` or `Kx` ?

Reply:
Oh, Kx then, I forgot the naming 😅